### ISSUE TITLE
sbt-release shouldn't push commits to origin repo

### ISSUE THREAD

**Issue by schmmd** [allenai/sbt-plugins#122]
When we release to Sonatype, the artifacts are not fully released until the artifacts are closed and released (a cancelable process requiring manual intervention).  Because of this, I don't think we should push commits to the origin repository automatically.

**Comment by dirkgr** [comment]
There is something else we could do. We can make the process automatic. There is an sbt plugin that does this, but it does not work right. It might be better to write our own, or clone the one that exists and fix it. The Sonatype web API doesn’t seem very difficult.

**Comment by schmmd** [comment]
What do you want to make automatic?  Closing and releasing the repositories?  That seems dangerous.  Those steps exist manually so you can verify your build before publishing it permanently.

**Comment by dirkgr** [comment]
That's true, but we don't actually verify anything, do we? Sonatype does a bunch of verification steps, and we just wait until it's done with those before we press the button.

As it is, it's more like a 24h waiting period to buy a gun, i.e., it gives you time to reconsider.

**Comment by schmmd** [comment]
I generally verify my build artifacts by adding the staging repository and
running a library that depends on them.

**Comment by dirkgr** [comment]
That's a lot of diligence. Making me feel like I'm just flying by the seat of my pants :-)

Will change it.

**Comment by schmmd** [comment]
It actually asks whether you want to push or not, so I'm not sure it requires changing.  I submitted this bug when I didn't have my branch set up to track, so it failed.

**Comment by dirkgr** [comment]
Oh, right. Then I’ll just change the instructions.
